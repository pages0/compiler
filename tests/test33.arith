let x: <int>  = ref 1+5 in
let y: <bool> = ref false in
x := !x + !x;
!x