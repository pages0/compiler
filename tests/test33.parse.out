( let [ x ( ref ( + 1 5 ) ) ] ( let [ y ( ref false ) ] ( ; ( != x ( + ( ! x ) ( ! x ) ) ) ( ! x ) ) ) )